src/components/pagination/pagination.tsx

```diff
@@ -292,8 +292,8 @@ export const EuiPagination: FunctionComponent<Props> = ({
 
       centerPageCount = (
         <ul
-          {...accessibleName}
           css={paginationStyles.euiPagination__list}
+          {...accessibleName}
           className="euiPagination__list"
```

Comment:
minor nit, let's keep `css` and `className` together for consistency whenever possible
```suggestion
          css={paginationStyles.euiPagination__list}
          className="euiPagination__list"
          {...accessibleName}
```

I will say though that this spread operator is kind of misleading, it's really only spreading a set of aria attributes and not `css` or any other props, so it's not really affecting anything. not a huge deal though